common/changes/@itwin/viewer-react/replaceTreeWidget_2021-08-09-19-51.json

```
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/viewer-react",
+      "comment": "Updated tree widget to use the UiProvider instead of the widget control",
+      "type": "patch"
```

let make this a minor bump here since we bumped the tree-widget ver
```suggestion
      "type": "minor"
```